iac/stacks/sagemaker.py

```diff
@@ -211,7 +250,7 @@ def _create_bucket(self, id: str, bucket_name: str) -> s3.Bucket:
             self,
             id=id,
             bucket_name=bucket_name,
-            versioned=False,
+            versioned=True,
```

Comment:
Does turning versioning on have any potential downsides? There's a possibility all public CoralNet data will get stuck in here -- will that e.g. increase costs?